src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -69,7 +70,7 @@ const styles: Styles<Theme, object> = {
 
 const genMapLink = (location: string) => {
     try {
-        const location_id = locations[location.split(' ')[0] as keyof typeof locations];
+        const location_id = locations[location.split(' ')[0]];
```

Comment:
```suggestion
        const location_id = locations[location.split(' ')[0] as keyof typeof locations];
```
if you want to keep typescript happy